### PR TITLE
Remove dev version of nodejs

### PR DESCRIPTION
**Pull request by ilgonmic**
Since 1.9.24 and 2.0.0 there is no necessity to set dev version

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -10,13 +10,6 @@ val deploy: Task? by tasks.creating {
...
 
-// Right now it is used for switching nodejs version which is supports generated wasm bytecode - Remove after updating to Kotlin 2.0
-extensions.findByType(NodeJsRootExtension::class.java)?.let {
-    // canary nodejs that supports recent Wasm GC changes
-    it.nodeVersion = "21.0.0-v8-canary202309167e82ab1fa2"
-    it.nodeDownloadBaseUrl = "https://nodejs.org/download/v8-canary"
-}
-
 // We need to ignore unsupported engines (i.e. canary) for npm
 tasks.withType(KotlinNpmInstallTask::class).configureEach {
```
Comment (dkrasnoff) [inline build.gradle.kts:14]: Can we also remove this?

Comment (ilgonmic) [reply build.gradle.kts:14]: Not sure for now, let's try

Comment (ilgonmic) [reply build.gradle.kts]: Ready

**Review by mvicsokolova — APPROVED** [review]

File: build.gradle.kts (hunk 1 above)
Comment (dkrasnoff) [reply build.gradle.kts]: Thanks!

**Review by dkrasnoff — APPROVED** [review]

**Pull request merged**